Domain/Sia.Shared/Extensions/linqExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace System.Linq
+{
+    public static class LinqExtensions
+    {
+        public static void Map<T>(this IEnumerable<T> input, Action<T> function)
```

Comment:
I don't see this extension method being used anywhere in this PR. 

Also, "map" functions usually transform the input collection into a new collection. But, this one doesn't return anything back.

This seems to replicate the functionality of IEnumerable.ForEach.

Reply:
There was a use in Sia.Gateway, but I've replaced that with ForEach. Removing this.